CONTRIBUTING.md

```diff
@@ -0,0 +1,295 @@
+# Introduction
+
+Well hello there my dear reader! Thank you for considering contributing to the Fantomas project!<br />
```

Comment:
I think there should be a comma after "there".  There should probably also be one after "Well", but I'm torn because it just starts being comma-heavy.